src/content/RepoPage/RepoPage.js

```diff
@@ -64,10 +79,116 @@ const RepoPage = () => {
       <div className="bx--row repo-page__r1">
         <div className="bx--col-lg-16">
           <RepoTable headers={headers} rows={rows} />
```

Comment:
The <Query> component should go here, replacing `<RepoTable headers={headers} rows={rows} />`